app/views/layouts/design_system.html.erb

```diff
@@ -0,0 +1,43 @@
+<% environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment %>
+
+<%= render "govuk_publishing_components/components/layout_for_admin", product_title: "Publisher", browser_title: yield(:page_title), environment: environment  do %>
+
+  <% render "layouts/google_tag_manager" %>
+
+  <%= render "govuk_publishing_components/components/skip_link" %>
+
+  <% user = current_user %>
+
+  <%= render "govuk_publishing_components/components/layout_header", {
+    product_name: "Publisher",
+    environment: environment,
+    navigation_items: navigation_items(current_user.govuk_editor?)
+  } %>  
+
+  <div class="govuk-width-container">
+    <% [:success, :info, :warning, :danger, :notice, :alert].select { |k| flash[k].present? }.each do |k| %>
```

Comment:
I don't think we want to use this for our flash messages—we would want to use components from the design system ([success](https://components.publishing.service.gov.uk/component-guide/success_alert), [error](https://components.publishing.service.gov.uk/component-guide/error_alert), [notice](https://components.publishing.service.gov.uk/component-guide/notice)).